tests/test_samples.py

```diff
@@ -600,7 +600,7 @@ def test_contour_plot_2d_nan():
     ns = NestedSamples(root='./tests/example_data/pc')
 
     ns.loc[:9, 'x0'] = np.nan
-    with pytest.raises((np.linalg.LinAlgError, RuntimeError)):
+    with pytest.raises((np.linalg.LinAlgError, RuntimeError, ValueError)):
```

Comment:
This is preferable to a more general `Exception`, since this way we will catch any changes which raise an exception for the wrong reason.